core/templates/dev/head/pages/suggestion_editor/ShowSuggestionModalForCreatorViewService.js

```diff
@@ -103,13 +125,16 @@ oppia.factory('ShowSuggestionModalForCreatorViewService', [
           };
         }
 
-        $http.put(url, data).then(function() {
+        $http.put(url, data).then(function(response) {
           for (var i = 0; i < suggestionsToReviewList.length; i++) {
             if (suggestionsToReviewList[i] === activeThread) {
               suggestionsToReviewList.splice(i, 1);
               break;
             }
           }
+          if (response) {
+            activeThread.suggestion.newValue = response.data.new_value;
```

Comment:
Why do you need the response? Can't you directly set it from the existing values in the data you sent in the put request?

Reply:
yes i can. Will change it

Reply:
done